test/Test/Core/Issue.hs

```diff
@@ -0,0 +1,52 @@
+module Test.Core.Issue where
+
+import IW.Core.Id (Id (..))
+import IW.Core.Issue (Issue (..))
+import IW.Core.SqlArray (SqlArray (..))
+
+import Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
+testLabels :: [Text]
+testLabels = [ "good first issue"
+             , "help wanted"
+             , "low hanging fruit"
+             , "docs"
+             , "easy"
+             ]
+
+genIssue :: MonadGen m => m Issue
+genIssue = do
+    id' <- genId
+    number <- genNumber
+    title <- genTitle
+    body <- genBody
+    repoOwner <- genRepoOwner
+    repoName <- genRepoName
+    labels <- genLabels
+    -- | Construct issue url
+    let url = "https://github.com/" <> repoOwner <> "/" <> repoName <> "/issues/" <> show number
+
+    pure $ Issue id' number title body url repoOwner repoName labels 
+    where
+        genId :: MonadGen m => m (Id Issue)
+        genId = Id <$> Gen.int (Range.constant 8 40)
+
+        genNumber :: MonadGen m => m Int
+        genNumber = Gen.int (Range.constant 8 40)
+   
+        genTitle :: MonadGen m => m Text
+        genTitle = Gen.text (Range.constant 8 40) Gen.alpha
+
+        genBody :: MonadGen m => m Text 
+        genBody = Gen.text (Range.constant 8 40) Gen.alphaNum
+
+        genRepoOwner :: MonadGen m => m Text
+        genRepoOwner = Gen.text (Range.constant 8 40) Gen.alpha
```

Comment:
I propose to start it from `1` because the owner can be one character long. Also, the generator should be `alphaNum` because user names can contain digits.